django/utils/_os.py

```diff
@@ -11,8 +11,8 @@ def safe_join(base, *paths):
     Join one or more path components to the base path component intelligently.
     Return a normalized, absolute version of the final path.
 
-    Raise ValueError if the final path isn't located inside of the base path
-    component.
+    Raise SuspiciousFileOperation if the final path isn't located inside of the base
+    path component.
```

Comment:
Oops I'll update my vimrc for Django 👍😅

Reply:
You don't need to, we use 79 chars limit only for comments and docstrings.

Reply:
Yep I saw that in .editorconfig. I need to update my vimrc because I have colorcolumn setup which is what I was using when I did the first change. When I checked just now it was using the 88 setting from .editorconfig. I've set it now to 79 for the Django dir.